JavaAssignments/src/amarja/AllDiffindex.java

```diff
@@ -0,0 +1,35 @@
+package amarja;
+import java.util.Scanner;
+//Question 9 Program to find all different index from given array
+public class AllDiffindex {
+	public static void main(String args[])
+	{		Scanner sc=new Scanner(System.in);//Reading the size and values of array 1
+			System.out.println("Enter size of first array:");
+			int size1=sc.nextInt();
+			System.out.println("Enter values of first array:");
+			int[]num1 = new int[size1];
+			for (int index=0;index<size1;index++)
```

Comment:
Amarja, here you wrote same logic two times to get data from user. Tomorrow if I asked to find differ index from 5 arrays then same logic you need to write for 5 times. always better to create function which will return int array of user given size.

Reply:
Incorporated review comments as suggested